contrib/vs2019/LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
```

Comment:
Do we really encourage the usage of individual licenses from contributors? I did not expect it and would recommend to remove this file.

Reply:
@tbeu I don't know, Thomas.  I've given up asking anybody to make any change at all.  It's pointless to ask.  @D4N has asked me to add a CONTRIBUTORS file to the repos to deal with this.  #1121

Reply:
I am not wedded to any particular license and am ok "subsuming" this into the overall license if fits better.

Reply:
Removed the LICENSE file if that's causing issues.